src/kaspastratum/stratum_server.go

```diff
@@ -21,6 +22,9 @@ type BridgeConfig struct {
 	PrintStats      bool   `yaml:"print_stats"`
 	UseLogFile      bool   `yaml:"log_to_file"`
 	HealthCheckPort string `yaml:"health_check_port"`
+	BlockWaitTime   string `yaml:"block_wait_time"`
```

Comment:
change this from a string to a time.Duration? It'll make life easier later on. You can parse it directly using flag.DurationVar

Reply:
sure... was just following existing style, and am also new to go, so didn't dive in too deep.  Will take a look.